From bff0c00e02594ea0c5413f5c53a8b2f8b2896d9d Mon Sep 17 00:00:00 2001
From: Byron Lee <byronlee@chromium.org>
Date: Tue, 14 Jan 2025 02:37:50 +0000
Subject: [PATCH] Add string_util include

The header is removed in crrev/c/6133861 and lots of clients using
string_util didn't include the header correctly.

Add the header back to fix the build error during uprev.

Change-Id: I8f6d554f2ff7416cb090941cbffec84a734133b7
---
 mojo/public/cpp/bindings/lib/string_serialization.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mojo/public/cpp/bindings/lib/string_serialization.h b/mojo/public/cpp/bindings/lib/string_serialization.h
index a8ff37d10d..7b666841f4 100644
--- a/mojo/public/cpp/bindings/lib/string_serialization.h
+++ b/mojo/public/cpp/bindings/lib/string_serialization.h
@@ -10,6 +10,7 @@
 
 #include <string_view>
 
+#include "base/strings/string_util.h"
 #include "mojo/public/cpp/bindings/lib/array_internal.h"
 #include "mojo/public/cpp/bindings/lib/message_fragment.h"
 #include "mojo/public/cpp/bindings/lib/serialization_forward.h"
-- 
2.47.1.688.g23fc6f90ad-goog

